bear/main.py.in

```diff
@@ -92,11 +92,13 @@ IGNORED_FLAGS = {
 
 }  # type: Dict[str, int]
 
+
 # Known C/C++ compiler wrapper name patterns.
 COMPILER_PATTERN_WRAPPER = re.compile(r'^(distcc|ccache)$')
 
 # Known MPI compiler wrapper name patterns.
-COMPILER_PATTERNS_MPI_WRAPPER = re.compile(r'^mpi(cc|cxx|CC|c\+\+)$')
+
```

Comment:
Is this change required? Can you provide me links for MPI compiler wrappers called `mpifort` or `mpif77`?

Reply:
@rizsotto I googled mpifort and found https://www.mpich.org/static/docs/v3.1.2/www1/mpifort.html . You should be able to find mpif77 the same way.